backend/RoomPlanner/room-planner/src/main/java/edu/roomplanner/service/impl/EncryptionServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package edu.roomplanner.service.impl;
+
+import edu.roomplanner.service.EncryptionService;
+import org.jasypt.util.text.BasicTextEncryptor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EncryptionServiceImpl implements EncryptionService {
+
+    @Value("${security.encryption.password}")
+    private String password;
+
+    public EncryptionServiceImpl() {
```

Comment:
If no other constructor with arguments is defined this declaration here is redundant.